server/bin/pbench-server-set-result-state

```diff
@@ -5,8 +5,8 @@
 opts=$SHELLOPTS
 case $opts in
     *xtrace*)
-        dir=$(dirname $(which $0))
-        PROG=$(basename $(which $0))
+        dir=$(dirname $(command -v $0))
+        PROG=$(basename $(command -v $0))
```

Comment:
Same comment as with `pbench-remote-sync-package-tarballs` regarding an assertion being added after line 15 below.

Reply:
And if you could add the same `case $# in/esac` statement as it exists in `pbench-remote-sync-package-tarballs`, that would be even better.